lms/envs/common.py

```diff
@@ -1917,6 +1922,15 @@
 CERT_NAME_SHORT = "Certificate"
 CERT_NAME_LONG = "Certificate of Achievement"
 
+#################### Badgr OpenBadges generation #######################
+BADGR_API_TOKEN = None
+BADGR_BASE_URL = "http://localhost:8005"
+BADGR_ISSUER_SLUG = "example-issuer"
+# A dictionary of course modes to file paths. Example:
+# {'honor': '/path/to/badge_image.png'}
+# Badge images must be square and in PNG or SVG format.
+BADGR_IMAGE_SOURCES = {}
```

Comment:
* Yes, it would be good to have defaults, so that it works better out of the box, without having to care about creating image badges initially.  It should be overridable though, as the default would likely be Open edX branded, while edx.org would have its own graphic I would imagine. @pbaruah Do we have image assets for the 3 badges already?
* Which raises the issue of updating the badge image when it is changed - especially the first time someone would try setting a badge, they might make a mistake - while we probably can't do much about badges that have already been generated, it would be tough to get stuck with the initial test image for all future badges generation. I don't see a command to update an existing badge in the API though (right?), so that would require recreating a new badge when it changes?
* Having a model makes it easier to have control over the file - besides ensuring that it is available at the time of the first request, it would also allow to check things like the size, or trigger a badge update when it is changed.